src/main/frontend/fs/diff_merge.cljc

```diff
@@ -0,0 +1,93 @@
+(ns frontend.fs.diff-merge
+  ;; Disable clj linters since we don't support clj
+  #?(:clj {:clj-kondo/config {:linters {:unresolved-namespace {:level :off}
```

Comment:
We can probably get rid of this, the nbb conditional and .cljc if this ns doesn't need to be nbb compatible

Reply:
Forgot what `cljc` stands for here; removing it 🤣